llvm/lib/Target/AMDGPU/AMDGPUISelLowering.cpp

```diff
@@ -3062,6 +3068,25 @@ static bool isCttzOpc(unsigned Opc) {
   return Opc == ISD::CTTZ || Opc == ISD::CTTZ_ZERO_UNDEF;
 }
 
+void AMDGPUTargetLowering::replaceCTLZResults(
```

Comment:
lowerCTLZ?

Reply:
I thought it might be confusing since we already have a `LowerCTLZ_CTTZ` function. Other backends use the "replaceXResults" convention which seemed appropriate. Or we could split the difference and call it `lowerCTLZResults`.